src/odemis/acq/test/move_test.py

```diff
@@ -329,6 +336,51 @@ def test_moving_to_grid1_in_sem_imaging_area_after_loading_1st_method(self):
         grid_label = self.posture_manager.getCurrentGridLabel()
         self.assertEqual(position_label, SEM_IMAGING)
         self.assertEqual(grid_label, GRID_1)
+        # check the values of tilt and rotation
+        sem_angles = self.stage.getMetadata()[model.MD_FAV_SEM_POS_ACTIVE]
+        if self.stage_version == "zeiss_1":
+            self.assertAlmostEqual(self.stage.position.value["rx"], sem_angles["rx"], places=5)
+            self.assertAlmostEqual(self.stage.position.value["rm"], sem_angles["rm"], places=5)
+        else:
+            self.assertAlmostEqual(self.stage.position.value["rx"], sem_angles["rx"], places=5)
+            self.assertAlmostEqual(self.stage.position.value["rz"], sem_angles["rz"], places=4)
+
+    # test linked ym axis movement when in fm imaging area
+    def test_moving_in_grid1_fm_imaging_area_after_loading(self):
+        """Check if the stage moves in the right direction when moving in the fm imaging grid 1 area."""
+        # move the stage to the loading position
+        f = self.posture_manager.cryoSwitchSamplePosition(LOADING)
+        f.result()
+        # move the stage to the fm imaging area, and grid1 will be chosen by default
+        f = self.posture_manager.cryoSwitchSamplePosition(FM_IMAGING)
+        f.result()
+        position_label = self.posture_manager.getCurrentPostureLabel()
+        self.assertEqual(position_label, FM_IMAGING)
+        # check the values of tilt and rotation
+        fm_angles = self.stage.getMetadata()[model.MD_FAV_FM_POS_ACTIVE]
+        if self.stage_version == "zeiss_1":
+            self.assertAlmostEqual(self.stage.position.value["rx"], fm_angles["rx"], places=5)
+            self.assertAlmostEqual(self.stage.position.value["rm"], fm_angles["rm"], places=5)
+        else:
+            self.assertAlmostEqual(self.stage.position.value["rx"], fm_angles["rx"], places=5)
+            self.assertAlmostEqual(self.stage.position.value["rz"], fm_angles["rz"], places=4)
+
+        # move in the same imaging mode using linked YM stage
+        old_stage_pos = self.stage.position.value
+        old_linked_ym_pos = self.linked_ym_stage.position.value
+        self.linked_ym_stage.moveRel({"y": 1e-3}).result()
+        new_stage_pos = self.stage.position.value
+        new_linked_ym_pos = self.linked_ym_stage.position.value
+
+        self.assertAlmostEqual(old_linked_ym_pos["y"] + 1e-3, new_linked_ym_pos["y"], places=5)
+        self.assertTrue(old_stage_pos["y"] < new_stage_pos["y"])
+
+        if self.stage_version == "zeiss_1":
+            # the stage moved in the right direction if the pre-tilt angle was maintained at -26-degrees
+            beta = 0.4537856055185  # -26-degrees in radians
```

Comment:
This is not -26° but +26°. And why not use math.radians for that? `beta = math.radians(26)`